### PR TITLE
Buckify breeze

### PR DESCRIPTION
**Pull request by Yuhta**
Differential Revision: D65797319

### CODE REVIEW

**Comment by aditi-pandit** [comment]
@Yuhta : Just curious. What is breeze ?

**Comment by Yuhta** [comment]
@aditi-pandit Breeze is a new library developed by Rivos for cross platform SIMT acceleration. We keep it inside Velox so that it can be co-evolved with Velox.

**Comment by Yuhta** [comment]
CC: @dreveman

File: velox/experimental/wave/common/tests/CMakeLists.txt
Comment (majetideepak) [inline velox/experimental/wave/common/tests/CMakeLists.txt:30]: Better to use a header-only library for CMake.
https://discourse.cmake.org/t/header-only-library/5476

Comment (Yuhta) [reply velox/experimental/wave/common/tests/CMakeLists.txt:30]: I will leave this to @dreveman to encapsulate the library in a cmake project

File: velox/experimental/breeze/breeze/platforms/openmp.h
Comment (dreveman) [inline velox/experimental/breeze/breeze/platforms/openmp.h:238]: can you move this change to a separate commit that has a short commit message explaining the need for it?

File: velox/experimental/breeze/breeze/utils/types.h
Comment (dreveman) [inline velox/experimental/breeze/breeze/utils/types.h:169]: same here. separate commit would be nice for this.

File: velox/experimental/breeze/test/generated/algorithms/algorithm_test-openmp.h
Comment (dreveman) [inline velox/experimental/breeze/test/generated/algorithms/algorithm_test-openmp.h:20]: Now that we're checking in this code. Let's make the generator code add a comment here that makes it clear that this file was auto-generated and should not be edited directly. Something like:

```
/*
 * This file is auto-generated from test/generate.sh
 * DO NOT EDIT!
 */
```

File: velox/experimental/wave/common/tests/CMakeLists.txt
Comment (dreveman) [reply velox/experimental/wave/common/tests/CMakeLists.txt:30]: Yep, sounds like a good thing to do as a follow up.

**Review by dreveman — APPROVED** [review]
lgtm. just some suggestions but they can also be done as follow ups

File: velox/experimental/breeze/test/generated/algorithms/algorithm_test-openmp.h
Comment (Yuhta) [reply velox/experimental/breeze/test/generated/algorithms/algorithm_test-openmp.h:20]: Sorry only saw this comment after merging, it's a good idea, can you add it to #11537?

**Pull request closed without merging**
